torchrl/objectives/td3.py

```diff
@@ -26,6 +26,25 @@
 )
 from torchrl.objectives.value import TD0Estimator, TD1Estimator, TDLambdaEstimator
 
+RANDOM_MODULE_LIST = (
```

Comment:
I expanded the list quite a bit to cover more modules. What do you think about how we should handle it? Importing the list from torchrl/objectives/utils.py?